cmd/apiserver/app/server/run_server.go

```diff
@@ -49,8 +49,10 @@ func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	glog.Infoln("Installing TPR types to the cluster")
 	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
 	if err := tprInstaller.InstallTypes(); err != nil {
-		glog.V(4).Infof("Installing TPR types failed, continuing anyway (%s)", err)
+		glog.Errorf("Failed to installing TPR types (%s)", err)
+		return err
```

Comment:
Done, I exported an error in the `tpr` package since I think it belongs there. The test is asserting on the error type now. https://github.com/kubernetes-incubator/service-catalog/pull/534/files#diff-022844fbd37a2a076952714e11067c85R58